README.md

```diff
@@ -184,11 +173,122 @@ Get the status of a job:
 
 ```js
 api.getJobStatus(jobId).then(function(status) {
-  pprint(status);
+  // Use status
 });
 ```
 
 
+## Application Quickstart
+
+This section provides a brief guide to using the Vision Services API with your
+application admin account.
```

Comment:
Same comment here. The wording reads as if a user is downloading a typical user token, rather than an application key/keyset.